ssot/gpu.py

```diff
@@ -1251,6 +1256,26 @@ class GpuGV100(GpuGP100):
     bios_major = 0x88
     bios_chip = 0x00
 
+# Turing
+
+class GpuTU102(GpuGV100):
+    gen = GenTuring
+    id = 0x162
+    pciid = 0x1e04  # also 0x1e07
```

Comment:
write 0x1e00 here -- the low pciid_varbits bits are variable anyhow

Reply:
Done